base/server/scripts/operations

```diff
@@ -50,13 +49,6 @@ case $command in
         # 1 generic or unspecified error (current practice)
         default_error=1
         ;;
-    stop|restart)
-        if $debian; then
-            default_error=1
-        else
-            default_error=2
-        fi
-        ;;
```

Comment:
I'm not familiar with this code, but assuming we just want to remove Debian-specific code, this code probably should remain:
```
stop|restart)
    default_error=2
    ;;
```

Reply:
as you can see from pkidaemon, stop/restart are only for debian